julia/src/visualize.jl

```diff
@@ -207,7 +207,9 @@ function _format_graphviz_edge(io::IOBuffer, head, tail, attrs)
   _format_graphviz_attr(io, attrs)
 end
 function _extract_shape(str :: AbstractString)
-  shape = matchall(r"\d+", str)
+  #shape = matchall(r"\d+", str)
```

Comment:
you can delete this

Reply:
Done.

Sorry for the mess, my first PR....